Evaluation/Input/Content/Section2.2_Data.md

```
@@ -0,0 +1,53 @@
+
+### Clinical Data  <a id="clinical-data"></a>
+
+A literature search was performed to collect available clinical data on levonogestrel in healthy adults.
+
+#### Model Building <a id="model-building"></a>
+
+The following studies were used for model building (training data):
+
+| Publication                 |Dose | Route| Arm / Treatment / Information used for model building |
+| :-------------------------- |---| -----|----------------------------------------------- |
+| [Cicali 2021](#main-references) | 0.09 mg       |IV    |Healthy women receiving a single dose administration of lenovogestrel 
```

markdownlint-cli2 (0.18.1)

Link fragments should be valid

(MD051, link-fragments)